cypress/integration/event-signup-flow.spec.js

```diff
@@ -717,4 +717,94 @@ describe("Verify page load " + new Date(), () => {
 				});
 			});
 	});
+	it('It nativates to selected "Online Events" page', function () {
+		cy.visit("/events/category/online-events", {
+			auth: {
+				username: Cypress.env("HTTPAUTH_USERNAME"),
+				password: Cypress.env("HTTPAUTH_PASSWORD"),
+			},
+		});
+		cy.acceptCookie();
+		cy.get(".events-featured__list__item")
```

Comment:
It took me a minute to figure out what was going on here - is there any way to make it clearer? there's so much nesting going on its hard to follow, but it boils down to:

```
event = getFirstEvent();
eventName = getName(event);
navigateTo(event);
pageHeading = getPageHeading();
expect(eventName).to equal(pageHeading);
```

We're also repeating this for each test, can we DRY it up?

Reply:
Resolved